modules/plot/main/coffee/Sparkline.coffee

```diff
@@ -37,7 +37,7 @@ class Sparkline
     hx.components.clear(selector)
     hx.component.register(selector, this)
 
-    graph = new hx.Graph(selector)
+    graph = new hx.Graph(selector, options)
```

Comment:
Ok. So I'll take that out again for now (just leaving the redrawOnResize option)